src/layout/Layout.jsx

```diff
@@ -1,15 +1,16 @@
 import * as React from "react";
```

Comment:
- Remove this line of code because it has no effect.
- Try to avoid importing `all` => `*` when you want to import so many things.

Reply:
Actually i dont know how its written 😅, i know react is global and now need to import.